src/routes.rs

```diff
@@ -39,6 +40,12 @@ lazy_static::lazy_static! {
     static ref START_TIME: u64 = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis() as u64;
 }
 
+thread_local! {
+    static LAST_DISPENSED: RefCell<HashMap<Address, tokio::time::Instant>> = RefCell::new(HashMap::new());
```

Comment:
It is a bad idea to use `RefCell` because if you try to modify it from several places, it will panic.

<img width="967" alt="image" src="https://github.com/FuelLabs/faucet/assets/18346821/9f7bc187-ce0c-413f-a2e5-703b3964bce3">